config/initializers/hoptoad.rb

```diff
@@ -1,29 +0,0 @@
-HoptoadNotifier.configure do |config|
-  # Change this to some sensible data for your errbit instance
-  config.api_key = ENV['OSEM_ERRBIT_APIKEY'] || Rails.application.secrets.errbit_key || ''
-  config.host    = ENV['OSEM_ERRBIT_HOST']
-  if config.api_key.blank? || config.host.blank?
-    config.development_environments = 'production development test'
-  else
-    config.development_environments = 'development test'
-  end
-
-  config.ignore_only = %w{
-    ActiveRecord::RecordNotFound
-    ActionController::InvalidAuthenticityToken
-    ActionController::UnknownAction
-    AbstractController::ActionNotFound
-    ActionView::MissingTemplate
```

Comment:
You forgot to transfer this to the new config file

Reply:
@AEtherC0r3 they are transferred in the current changeset... please update your review.